python/cudf/dataframe/dataframe.py

```diff
@@ -947,8 +947,25 @@ def _sanitize_columns(self, series):
         """Sanitize pre-appended
            col values
         """
-        if not isinstance(series, Series):
+
+        if (not isinstance(series, Series) and utils.is_list_like(series)) \
```

Comment:
utils.is_list_like(series) will always return false if the input dtype is a Series so you can eliminate this first condition entirely.

Reply:
eliminated it